terraform/modules/data-share-service/sns_alarm.tf

```diff
@@ -17,6 +19,46 @@ resource "aws_sns_topic" "sns_alarm_topic" {
   kms_master_key_id = aws_kms_key.sns_alarm_key.arn
 }
 
+data "aws_iam_policy_document" "sns_access" {
+  statement {
+    actions = [
+      "SNS:Subscribe",
+      "SNS:SetTopicAttributes",
+      "SNS:RemovePermission",
+      "SNS:Receive",
+      "SNS:Publish",
+      "SNS:ListSubscriptionsByTopic",
+      "SNS:GetTopicAttributes",
+      "SNS:DeleteTopic",
+      "SNS:AddPermission",
```

Comment:
This seems like lots of access, but I'm not too familiar with SNS permissions

Reply:
These are the default SNS permissions, I can change it so that our prometheus only has a subset?